data/de/lexicon.txt

```diff
@@ -0,0 +1,19 @@
+Januar;N;MASC;NOM;SG;INAN
```

Comment:
Is there a way to use some other format? This is the kind of format that I typically use.
```
Januar: singular accusative dative nominative masculine noun
Januare: plural accusative genitive nominative masculine noun
Januaren: plural dative masculine noun
Januars: singular genitive masculine noun
```
I omitted some mapping details between these entries, but that's mostly how I store this type of data. I also use an XML format that I convert to the syntax above. The XML structure is inverted to the style that I listed above.

Some other formats have been discussed in this group too.

As you expand to languages with numerous grammatical cases, it becomes hard to keep track of how these shouting abbreviations in uppercase map to a grammeme that has to be pronounced in a meeting or for a new person to interact with the data.

The nice feature about using XML is that it's easier to filter and annotate the data as necessary. This becomes important for languages that use bounded morphemes a lot (no whitespace separation) and use different values for the grammatical number for each morpheme.  For example, "your book" in Arabic is a single word with 2 morphemes.  The "your" can be singular, dual or plural. The "your" can be masculine or feminine for pronunciation, depending on who you're addressing. That possessive pronoun binds to the noun being possessed. The word "book" has a gender too, and it can be turned into singular, dual or plural.

Reply:
I was going for a simple format that's easy to prototype. Should we use XML from the start? We would need to agree on the basic structure. I'll change the structure now and then we can iterate.

Reply:
> Januar: singular accusative dative nominative masculine noun
> Januare: plural accusative genitive nominative masculine noun
> Januaren: plural dative masculine noun
> Januars: singular genitive masculine noun

Changed format to this for now. PTAL.

Reply:
Looks great to me! Thanks.

Reply:
Could you approve the PR?